### PR TITLE
Add Edit Project File command to menu Project in Nexus client.

### PR DESCRIPTION
**Pull request by ocallesp**
Fixes [AB#1218044](https://devdiv.visualstudio.com/0bdbc590-a062-4c3f-b0f6-9383f67865ee/_workitems/edit/1218044)

Add missing command.
image

### CODE REVIEW

File: src/Microsoft.VisualStudio.ProjectSystem.Managed.VS.Client/Menus.vsct [hunk 1]
```diff
@@ -69,6 +73,10 @@
     <GuidSymbol name="guidWorkspaceExplorerToolWindowPackageCmdSet" value="{cfb400f1-5c60-4f3c-856e-180d28def0b7}">
       <IDSymbol name="idmWSE_ContextMenu" value="0x0002"/>
     </GuidSymbol>
+
+    <GuidSymbol name="guidManagedProjectSystemClientMainMenuCmdSet" value="{d309f791-903f-11d0-9efc-00a0c911004f}">
+      <IDSymbol name="groupidCtxtMenuEditProjectFile" value="0x0237"/>
```
Comment (tmeschter) [inline src/Microsoft.VisualStudio.ProjectSystem.Managed.VS.Client/Menus.vsct:78]: Can you add a comment explaining where else these two values are defined?

**Review by jjmew — APPROVED** [review]
:shipit:

File: src/Microsoft.VisualStudio.ProjectSystem.Managed.VS.Client/Menus.vsct (hunk 1 above)
Comment (ocallesp) [reply src/Microsoft.VisualStudio.ProjectSystem.Managed.VS.Client/Menus.vsct:79]: done

**Comment by davkean** [comment]
This doesn't look correct. The shell has already placed these commands in the right places, they aren't showing up because the command hasn't been opt'd into Nexus (which you had a PR out for this here: https://devdiv.visualstudio.com/DevDiv/_git/VS/pullrequest/282545) and because we're not handling cmdidEditProjectFile: https://github.com/dotnet/project-system/blob/master/src/Microsoft.VisualStudio.ProjectSystem.Managed.VS.Client/SolutionExplorer/ProjectNodeCommandHandler.cs#L50.

File: src/Microsoft.VisualStudio.ProjectSystem.Managed.VS.Client/Menus.vsct [hunk 2]
```diff
@@ -69,6 +73,11 @@
     <GuidSymbol name="guidWorkspaceExplorerToolWindowPackageCmdSet" value="{cfb400f1-5c60-4f3c-856e-180d28def0b7}">
       <IDSymbol name="idmWSE_ContextMenu" value="0x0002"/>
     </GuidSymbol>
+
+    <!-- Guid and IDs copied from Visual Studio Shell -->
+    <GuidSymbol name="guidManagedProjectSystemClientMainMenuCmdSet" value="{d309f791-903f-11d0-9efc-00a0c911004f}">
```
Comment (davkean) [inline src/Microsoft.VisualStudio.ProjectSystem.Managed.VS.Client/Menus.vsct:78]: guidSHLMainMenu would be a better description of this. This GUID has nothing to do with us.

File: src/Microsoft.VisualStudio.ProjectSystem.Managed.VS.Client/Menus.vsct (hunk 1 above)
Comment (davkean) [reply src/Microsoft.VisualStudio.ProjectSystem.Managed.VS.Client/Menus.vsct:79]: Next time, please use the names as they are defined within the shell, for example, IDG_VS_CTXT_PROJECT_EDITFILE.

Comment (ocallesp) [reply src/Microsoft.VisualStudio.ProjectSystem.Managed.VS.Client/Menus.vsct:79]: I wanted to use the original name, but it was going to look different from the rest of the file.

**Pull request merged**